src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -1979,6 +2022,35 @@ def SetCheckMatchingFabric(self, check: bool):
             lambda: self._dmLib.pychip_DeviceController_SetCheckMatchingFabric(check)
         ).raise_on_error()
 
+    def EnableICDRegistration(self, parameters: typing.Optional[ICDRegistrationParameters] = None):
```

Comment:
Why allow "default' arguments? Suggest requiring explicit values to avoid "hidden" behavior